README.md

```diff
@@ -137,9 +142,7 @@ The example requires two markers: base (gme) and quote (usd).
         --home "$PIO_NODE" \
         --keyring-backend test \
         --chain-id "$CHAIN_ID" \
-        --gas auto \
-        --gas-prices 1905nhash \
-        --gas-adjustment 2 \
+        --fees 100381000000nhash \
```

Comment:
the gas fee we do here not enough for marker creation

Reply:
ya there is a flat fee for this now